### PR TITLE
Run RAS API tests with GC policy optthruput and a fixed 1500MB heap

### PR DESCRIPTION
**Pull request by keithc-ca**
Format consistently and replace
```
	<jvmarg value="-Xmx512M" />
```
with
```
	<jvmarg value="-Xgcpolicy:optthruput" />
	<jvmarg value="-Xms1024M" />
	<jvmarg value="-Xmx1024M" />
```

Fixes https://github.com/eclipse-openj9/openj9/issues/14193

### CODE REVIEW

File: test/functional/RasapiTest/test.xml [hunk 1]
```diff
@@ -42,53 +40,61 @@
...
-				<pathelement location="${ant.home}/lib/ant-launcher.jar"/>
+				<pathelement location="${ant.home}/lib/ant-launcher.jar" />
 			</classpath>
-			<jvmarg value="-showversion"/>
-			<jvmarg value="-Xmx512M"/>
-			<arg value="-buildfile"/>
-			<arg file="${ant.file}"/>
-			<arg value="test"/>
+			<jvmarg value="-showversion" />
+			<jvmarg value="-Xgcpolicy:optthruput" />
```
Comment (pshipton) [inline test/functional/RasapiTest/test.xml:55]: This one is the ant processing, it doesn't really need to change. Even the original value -Xm128m is fine here. On the other hand, it's not harmful to change it, this process doesn't (shouldn't) create any core files.

**Comment by pshipton** [comment]
Testing via grinder https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/619

**Comment by pshipton** [comment]
The grinder still failed due to `OutOfMemoryError: Java heap space` in testSystemDumpNullFile which is part of DumpAPIBasicTests.

**Comment by pshipton** [comment]
If 1500M works we could use that and be done with it.

File: test/functional/RasapiTest/test.xml (hunk 1 above)
Comment (keithc-ca) [reply test/functional/RasapiTest/test.xml:55]: Thanks for the reminder. I think this should be reverted to the original heap size (128M).

**Comment by keithc-ca** [comment]
Use 128MB to run `ant` and 1500MB for RAS API tests.

**Comment by pshipton** [comment]
https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/629 - killed due to timeout. We could try something between 1024M and 1500M. I'm going to pursue removing the debug image installation by default.

**Comment by keithc-ca** [comment]
Reduced heap from 1500MB to 1200MB.

**Comment by pshipton** [comment]
Started a new grinder https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/632/

The other idea I had was maybe we need to add some System.gc() calls to the test.

**Comment by pshipton** [comment]
Another timeout.

**Comment by pshipton** [comment]
Maybe we should just exclude the test on AIX?

**Comment by pshipton** [comment]
I suppose we can increase the timeout period.

**Comment by keithc-ca** [comment]
I doubled the timeout from 15 to 30 minutes.

**Comment by pshipton** [comment]
New grinder https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/639/

**Comment by pshipton** [comment]
The one passed.

**Review by pshipton — APPROVED** [review]

**Pull request merged**